config/manifests/bases/apicast-operator.clusterserviceversion.yaml

```diff
@@ -41,10 +41,10 @@ spec:
 
     ### Upgrading your installation
     The APIcast Operator understands how to run and upgrade between a set of APIcast versions.
-    See [the upgrade guide](https://github.com/3scale/apicast-operator) for more information.
+    See [the upgrade guide](https://github.com/3scale/apicast-operator/blob/v0.8.0/doc/operator-user-guide.md#upgrading-APIcast) for more information.
 
     ### Documentation
-    Documentation can be found on our [website](https://github.com/3scale/apicast-operator).
+    Documentation can be found on our [website](https://github.com/3scale/apicast-operator/tree/v0.8.0).
```

Comment:
same as above

Reply:
Thank you for comment. Yes, I will recheck this llink after tagging, and will confirm.